source/helpers/components/index.ts

```diff
@@ -0,0 +1,58 @@
+import { highlight, languages } from 'prismjs';
+import PrismLoader from 'prismjs/components/index';
+
+import { liquidEngine } from '../engines';
+import { default as codeExample } from './codeExample';
+import { compatibility, implStatus } from './compatibility';
+
+export { codeExample };
+export { compatibility, implStatus };
+
+/**
+ * Returns HTML for a fun fact that's not directly relevant to the main
+ * documentation.
+ */
+export const funFact = async (contents: string) =>
+  liquidEngine.renderFile('fun_fact', {
+    contents,
+  });
+
+/**
+ * Returns HTML for a code block with syntax highlighting via [Prism][].
+ *
+ * This should be equivalent to the [11ty `{% highlight %}` tag][hl-tag], except
+ * this tag can wrap dynamic content (partials, variables, etc), while the 11ty
+ * tag only wraps plain text.
+ *
+ * [Prism]: https://prismjs.com/
+ * [hl-tag]: https://www.11ty.dev/docs/plugins/syntaxhighlight/#usage
+ *
+ * @see https://prismjs.com/
+ */
+export const codeBlock = (contents: string, language: string, padding = 0) => {
+  if (!languages[language]) {
+    PrismLoader(language);
+  }
+  const code = `${contents}${'\n'.repeat(padding + 1)}`;
```

Comment:
This padding isn't quite right when there are multiple stacked blocks... I'll take a look.

Reply:
Oh, it's also slightly off in production: https://sass-lang.com/guide#topic-5

I'll take a look and see if I can find a quick fix...

Reply:
Fixed in https://github.com/oddbird/sass-site/commit/8d958ce4de17564aef0931f178f2c840a7336fd9